crates/rspack_regex/Cargo.toml

```diff
@@ -8,6 +8,7 @@ version    = "0.1.0"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+regex-syntax = "0.7.1"
```

Comment:
You may want to turn off the `unicode` feature

Reply:
Done